applications/openshift/master/file_permissions_multus_conf/oval/shared.xml

```diff
@@ -0,0 +1,27 @@
+<def-group>
+    <definition class="compliance" id="file_permissions_multus_conf" version="1">
+      {{{ oval_metadata(description="This test makes sure that ^/var/run/multus/cni/net.d/.*$ has mode 0644. If the target file or directory has an extended ACL, then it will fail the mode check.") }}}
```

Comment:
Please come up with a more compliance-like statement here for why this checks exists. For example:

```Ensure all files in /var/run/multus/cni/net.d/ have permissions 0644 or stricter.```

Reply:
That comes from the rule.yml fields. https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/master/file_permissions_multus_conf/rule.yml and are being populated by the template. This pull request is going to be replaced by https://github.com/ComplianceAsCode/content/pull/6476